front-end/src/App.js

```diff
@@ -1,11 +1,10 @@
 import { useState } from 'react';
 import './App.css';
```

Comment:
looks like this change could cause a conflict

import Registration from "./components/layout/SignUp";
import Landing from "./components/layout/Landing";
import Login from "./components/layout/LoginPage";